Source/TickerCounter.swift

```diff
@@ -105,17 +184,31 @@ public final class TickerCounter: UIView {
         guard let value = value else {
             return
         }
-        
         numbersText = value.description.flatMap{ String($0) }
     }
     
     private func createScrollLayers() {
-        let width = frame.width / CGFloat(numbersText.count)
-        let height = frame.height
+        guard let font = font else { return }
+        var characterSize = NSString(string: "0").size(withAttributes: [NSAttributedStringKey.font : font])
```

Comment:
id typically move this to a `String` extension - i.e. "0".size(whenFontApplied: font)